### DISCUSSION TITLE
Output format for storing and analysing large data

### DISCUSSION THREAD

**Discussion by MaheshTummala17** [idaholab/moose#28727] [Show and tell]
I am working on a model containing nearly 600k elements and storing the different State variable values at ten sync times. For now, I am outputting the data as a CSV, but handling such large data using Excel sheets and importing data via MATLAB/python for plotting is taking a significant amount of time. Which is the better format to store such large data sets? Does MOOSE support HDF5 format?

**Comment by GiudGiud** [comment]
Hello

CSV is likely preferred, a couple of millions of data points should not be unreasonable to work on from Python

But if you want to try other techniques, the exodus output can be read in Python using vtk utilities

Guillaume
